packages/adapter-mongoose/README.md

```diff
@@ -27,16 +27,12 @@ keystone.connect(mongoDbUri, mongooseOptions);
 
 ### `mongoDbUri`
 
-_**Default:**_ `'mongodb://localhost:27017/'`
+_**Default:**_ `'mongodb://localhost:27017/${SAFE_KEYSTONE_NAME}'`
```

Comment:
Maybe `${PUT_A_UNIQUE_DBNAME_HERE}` instead?

Reply:
I was trying to convey that it gets the value you give for `name` in your Keystone config and makes it safe to be used as a DB name. The behaviour before was the same, it just used 2 vars.